app/uk/gov/hmrc/helptosavefrontend/views/core/confirm_check_eligibility.scala.html

```diff
@@ -26,11 +26,18 @@
 ) {
   <h1 class="heading-large">@Messages("hts.no-account.title.h1")</h1>
 
-  <p class="lede">@Html(Messages("hts.no-account.p1"))</p>
+  <p>@Html(Messages("hts.no-account.p1"))</p>
```

Comment:
is there a reason you got rid of the `lede` class here?

Reply:
c.f. https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/Lead_paragraph.html

Reply:
Because the size of the text in the lede class doesn't match the new content

Reply:
I think we should check this with Nick and Jean

Reply:
I have just checked this with Nick and he has asked to put it back, so raising another PR